src/pages/about.astro

```diff
@@ -0,0 +1,300 @@
+---
+import '@styles/about.css';
+import { SITE_TITLE, SITE_DESCRIPTION } from 'src/consts';
+import Layout from '@layouts/Default.astro';
+---
+
+<Layout title={SITE_TITLE} description={SITE_DESCRIPTION}>
+	<!-- TypeIt Library -->
+	<script src="https://unpkg.com/typeit@8.7.1/dist/index.umd.js"></script>
```

Comment:
If typeit is already included in package.json, we can import it instead of downloading the library from unpkg

Reply:
How can I include typeit in package.json?